buildkite.yml

```diff
@@ -1,3 +1,5 @@
 steps:
+  - label: ':go:'
+    command: find internal/testdata -type f -name '*.go' -exec go fmt {} \;
```

Comment:
Would go fmt ./... be as sufficient?

Reply:
Nope, `go fmt ./...` excludes testdata directories! :(

Reply:
TIL

Reply:
Just to clarify, is the return value of this non-zero whenever there's a change on disk?

Reply:
oops, good catch - it's not. Will fix.